### DISCUSSION TITLE
A Question About Temporal RSA

### DISCUSSION THREAD

**Discussion by luonnyS** [rsagroup/rsatoolbox#386] [Q&A]
Hello
    I am a beginner about RSA analysis. And I am confused when I learned Temporal RSA Demo .
    For example, my time series is [0s, 1s, 2s, 3s, 4s, 5s, 6s], and when the time point is 3, what  does the corresponding RDM mean? It means time bin [2s, 3s] or all time points from 0s to 3s?
    I tried to figure out from source code but failed, sry for that.

**Comment by HeikoSchuett** [comment]
Hi @luonnyS,

the temporal RSA code computes RDMs per time point, i.e. the RDM at timepoints 3s is based on the data you had for 3s in the original time series. It does not accumulate anything over time. if you want to do that, you need to do the transformations to the time series before you pass it into the RDM calculation function.

Does that answer your question?

**Comment by JasperVanDenBosch** [comment]
To add to @HeikoSchuett 's answer, of course if you have done time binning on the Dataset, with `TemporalDataset.bin_time()`; the "timepoints" then refer to the bins.


Maybe you can show us some of our code to better understand your situation?
